examples/UKCore-Bundle-BatchPut-Example.xml

```diff
@@ -0,0 +1,40 @@
+<Bundle xmlns='http://hl7.org/fhir'>
+<id value="UKCore-Bundle-BatchPut-Example" />
+  <type value='batch'/>
+  <entry>
+    <resource>
+      <Parameters>
+        <parameter>
+          <name value='system'/>
+          <valueUri value='http://loinc.org'/>
+        </parameter>
+        <parameter>
+          <name value='code'/>
+          <valueCode value='23245-4'/>
```

Comment:
We don't really use LOINC in the UK, so this is bad example

Reply:
Actually , i have just post the example sample right now still have to work on it to make it Ukfied.